implementation-jaxrs2/src/main/java/io/smallrye/opentelemetry/jaxrs2/server/ServerTracingInterceptor.java

```diff
@@ -0,0 +1,41 @@
+package io.smallrye.opentelemetry.jaxrs2.server;
+
+import static io.smallrye.opentelemetry.jaxrs2.server.internal.SpanWrapper.PROPERTY_NAME;
+
+import java.util.List;
+
+import javax.annotation.Priority;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Priorities;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.ext.InterceptorContext;
+
+import io.opentelemetry.trace.Tracer;
+import io.smallrye.opentelemetry.jaxrs2.serialization.InterceptorSpanDecorator;
+import io.smallrye.opentelemetry.jaxrs2.serialization.TracingInterceptor;
+import io.smallrye.opentelemetry.jaxrs2.server.internal.CastUtils;
+import io.smallrye.opentelemetry.jaxrs2.server.internal.SpanWrapper;
+
+@Priority(Priorities.ENTITY_CODER)
+public class ServerTracingInterceptor extends TracingInterceptor {
+    /**
```

Comment:
What implementation is there that's using CXF?

Reply:
This instrumenation is based on https://github.com/opentracing-contrib/java-jaxrs that was compatible and tested against RestEasy, Jersey and Apache CXF. 

Once this POC is completed we could consider moving the JAX-RS instrumentation to OpenTelemetry project and just reuse it here like we did in SmallRye-OpenTracing.

Reply:
Also if I am not mistaken OpenLiberty uses CXF and they probably would like to use this implementation for tracing.

Reply:
"probably would like" is not actual usage.

Can you confirm with them whether they would use this. If they won't, I'd prefer we remove any CXF handling